variables.tf

```diff
@@ -70,6 +70,14 @@ variable "tags" {
   default     = {}
 }
 
+variable "abac_tags" {
+  type        = map(string)
+  description = "Map of tags that will be used inside IAM Policies for restricting EMR Service Role access"
+  default = {
+    "tamr.com/role" = "emr"
+  }
+}
```

Comment:
We need to decide whether we are going to set any tags by default. We want permissions to be as tight as possible out of the box, but we risk breaking things if the user forgets to tag the EMR subnet accordingly.